docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -497,7 +497,12 @@ following properties:
 * - `iceberg.rest-catalog.vended-credentials-enabled`
   - Use credentials provided by the REST backend for file system access.
     Defaults to `false`.
-  :::
+* - `iceberg.rest-catalog.case-insensitive-name-matching`
+  - Support case-insensitive namespace, table, view names. Defaults to `false`.
```

Comment:
```suggestion
  - Support case-insensitive namespace, table, and view names. Defaults to `false`.
```

also .. what about other objects like materialized views?

Reply:
MV is not supported for REST catalog as of yet. https://github.com/trinodb/trino/blob/master/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoRestCatalog.java#L547-L592